base_user_role/views/role.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2014 ABF OSIELL <http://osiell.com>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+
+<record id="view_res_users_role_form" model="ir.ui.view">
```

Comment:
Shouldnot this level be indented?

Reply:
@elicoidal  See [above comment](https://github.com/OCA/server-backend/pull/1#discussion_r151944395) of @sebalix

Reply:
OK so we are in line :smile: